examples/plotter/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "plotter"
+version = "0.1.0"
+authors = ["SixtyFPS <info@sixtyfps.io>"]
+edition = "2018"
+publish = false
+license = "GPL-3.0-only"
+
+[[bin]]
+path = "main.rs"
+name = "plotter"
+
+[dependencies]
+sixtyfps = { path = "../../api/sixtyfps-rs" }
+#plotters = { version = "0.3.1", default-features = false, features = ["bitmap"] }
+plotters = { version = "0.3.1", default-features = false, features = ["bitmap_backend", "surface_series"] }
+imgref = "1"
+rgb = "0.8.27"
```

Comment:
Would be nice if our users didn't have to import that. Could we not re-export it

Reply:
We could do that, yes. It's only needed for https://docs.rs/rgb/0.8.27/rgb/trait.ComponentBytes.html#method.as_bytes right now.

What's the best way of doing that?

Another option that doesn't require a re-export:

Currently the use is to get a (mutable) reference to a `[u8]` slice via these two methods. So it looks like this:

```rust
pixel_buffer
    .as_slice() // we provide that, to give access to the SharedVector as slice
    .as_bytes(); // provided by rgb's impl of ComponentBytes for the [PixelType]
```

So we could simply offer `as_bytes()` and `as_bytes_mut()` in the API and that's it:

```rust
pixel_buffer
    .as_bytes();
```

Reply:
I'll go ahead and also do that as a commit on top.